qa/common/shm_util.py

```diff
@@ -55,7 +55,10 @@ def _prepend_string_size(input_values):
             # don't convert it to str as Python will encode the
             # bytes which may distort the meaning
             if obj.dtype.type == np.bytes_:
-                s = bytes(obj)
+                if type(obj.item()) == bytes:
+                    s = obj.item()
+                else:
+                    s = bytes(obj)
```

Comment:
you forgot to clean up here and above in infer_util.py ...

Reply:
Fixed